impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -319,6 +359,53 @@ else if (this.clientConfig.getClientCredentialsResolver() == null) {
             this.clientConfig.setBaseUrlResolver(new DefaultBaseUrlResolver(this.clientConfig.getBaseUrl()));
         }
 
+        // OAuth2
+        if (this.clientConfig.getAuthenticationScheme() == AuthenticationScheme.OAUTH2) {
+            if (this.clientConfig.getAuthorizationMode() != null) {
+                if (this.clientConfig.getAuthorizationMode().equals("PrivateKey")) {
```

Comment:
For now `PrivateKey` should be the default value, as we only have one, so I'm guessing this should be something like `== null/empty || == "PrivateKey"`

Reply:
noted and addressed.